plone/app/layout/viewlets/document_byline.pt

```diff
@@ -52,20 +46,6 @@
   </span>
   </tal:dates>
 
-  <tal:expired tal:condition="view/isExpired">
-    —
-    <span class="state-expired"
-          i18n:translate="time_expired">expired</span>
-  </tal:expired>
```

Comment:
Is this information available in the toolbar?
This might also interesting for anonymous users...